script.js

```diff
@@ -0,0 +1,36 @@
+//script
+
+//get querystring
+
+const params = new URLSearchParams(window.location.search);
+
+if (params.has('version')) {
+  if (params.get('version') === 'a') {
+    version = 'a';
```

Comment:
Where is this variable declared. If there is no declaration it's like writing window.version which is a global variable and can leak.